programs/CMakeLists.txt

```diff
@@ -119,4 +119,9 @@ foreach (source_file ${check_programs})
 		"usrsctp"
 		${CMAKE_THREAD_LIBS_INIT}
 	)
+
+	if(APPLE)
+		set_target_properties(${source_file_we} PROPERTIES
+			XCODE_ATTRIBUTE_PRODUCT_BUNDLE_IDENTIFIER com.github.sctplab.usrsctp.${source_file_we})
+	endif()
 endforeach ()
```

Comment:
Committed in https://github.com/sctplab/usrsctp/commit/d52bb17c74b21096c9b205fa55dc6d0573cdc9a2 with whitespace changes.